src/App/App.tsx

```diff
@@ -2,13 +2,15 @@ import { Navbar } from "@components/Navbar";
 import { ROUTES } from "@configs/routes";
 import { ProductDetail } from "@pages/ProductDetail";
 import Products from "@pages/Products";
+import { useCartStoreInit } from "@store/RootStore/hooks/useCartStoreInit";
 import { useQueryParamsStoreInit } from "@store/RootStore/hooks/useQueryParamsStoreInit";
```

Comment:
можно сделать в папке @store/RootStore/hooks index-файл с реэкспортом всех хуков, чтобы здесь писать вот так:
```suggestion
import { useCartStoreInit, useQueryParamsStoreInit  } from "@store/RootStore/hooks";
```

просто короче и красивее

просто к твоему сведению